linera-core/src/client/mod.rs

```diff
@@ -83,6 +83,11 @@ mod chain_state;
 #[path = "../unit_tests/client_tests.rs"]
 mod client_tests;
 
+/// The maximum size of a data or bytecode blob, in bytes.
+pub(crate) const MAXIMUM_BLOB_SIZE: u64 = 3 * 1024 * 1024;
+/// The maximum size of decompressed bytecode, in bytes.
+pub(crate) const MAXIMUM_BYTECODE_SIZE: u64 = 30 * 1024 * 1024;
```

Comment:
Maybe extract `1024 * 1024` as `MEGABYTE`.

30MB for bytecode is huge, isn't it? What kind of bytecode is it? Contracts?

Reply:
You mean `MEBIBYTE`! :nerd_face: Will do!

The limit applies to both contracts and services, yes.
I'm not sure about it: I don't think there's a good reason to restrict it to less than that? It's only to defend against zip bombs.

Reply:
Yes this temporary.